src/Goteo/Model/LegalEntity.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+namespace Goteo\Model;
+
+use Goteo\Library\Text;
+
+use Goteo\Core\Model;
+use Goteo\Model\LegalEntity\NaturalPerson;
+use Goteo\Model\LegalEntity\LegalPerson;
+
+abstract class LegalEntity {
+
+    const NATURAL_PERSON = 'natural_person';
+    const LEGAL_PERSON = 'legal_person';
+    const LEGAL_ENTITIES = [self::NATURAL_PERSON, self::LEGAL_PERSON];
+
+    private string $legal_entity;
+
+    public static function create(string $legal_entity): LegalEntity {
+        $legal_entity_type;
+
+        switch ($legal_entity) {
+            case self::LEGAL_PERSON:
+                $legal_entity_type = new LegalPerson();
+            case self::NATURAL_PERSON:
+                $legal_entity_type = new NaturalPerson();
+            default:
+                $legal_entity_type = new NaturalPerson();
+        }
+
+        return $legal_entity_type;
+    }
```

Comment:
The above method could be simplified to:

public static function create(string $legal_entity): LegalEntity {
        switch ($legal_entity) {
            case self::LEGAL_PERSON:
                return new LegalPerson();
            default:
                return new NaturalPerson();
        }
    }